app/views/layouts/_feedback.html.slim

```diff
@@ -0,0 +1,56 @@
+- if params[:controller] == "results" || params[:controller] == "controlled_work_document_selections"
+  .gem-c-feedback__prompt
+    .govuk-width-container data-module="question-area"
+      span.gem-c-feedback__prompt-question-answer role="form"
+        h2.govuk-heading-s class="govuk-!-margin-bottom-0 govuk-!-margin-right-6"
+          = t("feedback.satisfaction.question")
+
+        = form_with(url: feedbacks_path(assessment_code: params[:assessment_code],
+                                        type: "satisfaction",
+                                        satisfied: "yes"),
+                                        local: false) do |form|
+          = form.govuk_submit "Yes", class: "gem-c-feedback__prompt-link-small-button", id: "yesButton"
+
+        = form_with(url: feedbacks_path(assessment_code: params[:assessment_code],
+                                        type: "satisfaction",
+                                        satisfied: "no"),
+                                        local: false) do |form|
+          = form.govuk_submit "No", class: "gem-c-feedback__prompt-link-small-button", id: "noButton"
+
+    .govuk-width-container data-module="link-area" hidden=true
+      span.gem-c-feedback__prompt-question-answer
+        h2.govuk-heading-s class="govuk-!-margin-bottom-0 govuk-!-margin-right-2"
+          = t("feedback.satisfaction.thank_you_message")
+        p.govuk-body-m class="govuk-!-margin-bottom-0 govuk-!-margin-right-6"
+          = t("feedback.satisfaction.feedback_link_html")
+- else
+  .gem-c-feedback__prompt data-module="freetext-feedback-area"
+    .govuk-width-container data-module="prompt-area"
+      .gem-c-feedback__prompt-content
+        .gem-c-feedback__prompt-questions--something-is-wrong
+          button.gem-c-feedback__prompt-link data-freetext-feedback="query"
```

Comment:
"query" isn't the clearest name to me. In technical terms, we have SQL queries and querystrings. In common parlance, a query is a question. But this button doesn't display a question, it displays a statement, and in particular I think it's a _prompt_ (it's in the prompt area after all).